content/member/eflahaut.yaml

```
@@ -0,0 +1,25 @@
+active: true
+anonymousTribePicture: true
+
+name: Emmanuelle Flahaut
+gender: female
+position: "Alternante Experte en stratégie digitale"
+bio: |
+    Après un master en management avec une spécialisation en communication, Emmanuelle a voulu changer de direction et s’orienter vers la direction artistique et le graphisme.De nouveau sur les bancs de l’école, elle voudrait devenir une experte capable d’intervenir autant sur les aspects stratégiques que graphique/design des projets qui lui sont confiés ✨🎨💻
+
+    Sinon, elle aime se creuser les méninges dans des escape games, les jeux vidéo, écrire toutes les histoires qui lui passent par la tête, la boxe française et le chant !
+
+integrationDate: 15-08-2023
+
+linkedIn: https://www.linkedin.com/in/emmanuelle-flahaut-035570b4/
```

Seul le pseudo / slug est nécéssaire:

```suggestion
linkedIn: emmanuelle-flahaut-035570b4/
```